data/exchange_rate_graph_helper.py

```diff
@@ -0,0 +1,79 @@
+from dateutil import relativedelta
+import pandas as pd
+
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+
+from data.currencies import Currencies
+from data.exchange_rate_graph_consts import ExchangeRateGraphConsts
+
+
+class ExchangeRateGraphHelper:
+    @staticmethod
+    def process_graph_month_format(date_of_today, currency_code, currency_exchange_rate_df):
+        start_date = pd.to_datetime(date_of_today - relativedelta.relativedelta(months=1))
+        graph = ExchangeRateGraphHelper.create_graph(start_date=start_date,
+                                                     end_date=date_of_today,
+                                                     currency_code=currency_code,
+                                                     df=currency_exchange_rate_df)
+        graph.xaxis.set_major_locator(mdates.WeekdayLocator(byweekday=mdates.SU))
+        ExchangeRateGraphHelper.show_graph()
+
+    @staticmethod
+    def create_graph_week_format(date_of_today, currency_code, currency_exchange_rate_df):
+        start_date = pd.to_datetime(date_of_today - relativedelta.relativedelta(weeks=1))
+        graph = ExchangeRateGraphHelper.create_graph(start_date=start_date,
+                                                     end_date=date_of_today,
+                                                     currency_code=currency_code,
+                                                     df=currency_exchange_rate_df)
+        graph.xaxis.set_major_locator(mdates.DayLocator())
+        ExchangeRateGraphHelper.show_graph()
```

Comment:
these function are  the same except for one variable. Combine them to one (change the relativedata field only)